test/testcafe/spec-en-leaks/EnglishLeaks_spec.js

```diff
@@ -19,7 +19,6 @@ const ignoredMocks = [
   'identify-with-apple-redirect-sso-extension.json', // flaky on bacon
   'identify-unknown-user.json',
   'error-user-is-not-assigned.json',
-  'error-internal-server-error.json',
```

Comment:
The ticket seems to be for fixing the message `Internal Server Error ` that is in https://github.com/okta/okta-signin-widget/blob/master/playground/mocks/data/idp/idx/error-internal-server-error.json#L7

`errors.E0000010` this key maps to different message in the login bundle  https://github.com/okta/okta-signin-widget/blob/master/packages/@okta/i18n/src/properties/login.properties#L73

Reply:
Yes, [`E0000009(Internal Server Error)`](https://github.com/okta/okta-signin-widget/blob/master/playground/mocks/data/idp/idx/error-internal-server-error.json#L7) is fixed by re-using existing(v1) [error message](https://github.com/okta/okta-signin-widget/blob/master/packages/%40okta/i18n/src/properties/login.properties#L72)
The mapping change(errors.E0000010 => E0000010) is there to make sure correct `E0000010` message is displayed for v2 as the messages diverge between v1 and v2 for `E0000010` but are the same for `E0000009`.